build.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/python3
+
+# This script generates the documentation for all the OpenWISP versions
+# defined in config.yml. For each version, it will clone or update the
+# OpenWISP modules and checkout the specified branch. It will then generate
+# the documentation using Sphinx in a separate directory for each
+# OpenWISP version.
+
+import os
+import subprocess
+import sys
+
+import yaml
+from jinja2 import Environment, FileSystemLoader
+
+OUTPUT_FORMATS = ['pdf', 'epub', 'html']
+
+
+def clone_or_update_repo(module_name, branch):
+    """
+    Clone or update a repository based on the module name and branch provided.
+    If the repository already exists, update it. Otherwise, clone the repository.
+    """
+    repo_url = f'https://github.com/openwisp/{module_name}.git'
```

Comment:
I was thinking here, since our repos are open source and public, I think we can use the git protocol here instead of SSH, isnt't it?

This will make it easier for us to work on the entire set of documents from one place and push changes to all the modules more easily. Now I found myself having to use `git remote set-url origin ...` a lot before being able to push.